test/e2e/core.go

```diff
@@ -97,3 +123,29 @@ func logCore(cmds []command, hosts []string, dir, provider string) {
 	}
 	wg.Wait()
 }
+
+func findKubeletSystemdService(provider string, hosts ...string) (bool, error) {
```

Comment:
two minor nits:
* maybe "isUsingSystemdKubelet" is a better name for this function?
* you never use the error return value - should this return only bool?